cryptorand/cipher.go

```diff
@@ -0,0 +1,48 @@
+package cryptorand
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+
+	"golang.org/x/xerrors"
+)
+
+type Cipher interface {
+	Encrypt([]byte) ([]byte, error)
+	Decrypt([]byte) ([]byte, error)
+}
+
+// CipherAES256 returns a new AES-256 cipher.
+func CipherAES256(key []byte) (Cipher, error) {
+	block, err := aes.NewCipher(key)
```

Comment:
the key should be verified to ensure it's 32 bytes for AES-256

Reply:
aes.NewCipher throws an error if it's not

Reply:
aes.NewCipher accepts smaller key sizes for less secure AES variants, we should just enforce 32 bytes for AES-256 which is the most secure algo supported

Reply:
Ahh, good catch